server/auth/routes.ts

```diff
@@ -203,3 +203,40 @@ auth.get('/logout', async (req, res) => {
 
 	res.redirect(new URL('/dashboard/login', process.env.APP_URL).toString());
 });
+
+/*
+ * If no organization exists:
+ * Step 3: create a new organization for the user
+ * B2B apps are org-first, so users MUST create or join an organization as part
+ * of signup.
+ */
+
+auth.post('/register', async (req, res) => {
+	const stytch = loadStytch();
+	const token = req.cookies.intermediate_token;
+	const organization = req.body.organization;
+	const slug = organization
+		.trim()
+		.toLowerCase()
+		.replace(/[\s+~\/]/g, '-')
+		.replace(/[().`,%·'"!?¿:@*]/g, '');
+
+    const result = await stytch.discovery.organizations.create({
+		intermediate_session_token: token,
+		organization_name: organization,
+		organization_slug: slug,
+	});
+
+	// TODO add the required resources and roles to the new organization
```

Comment:
@jlengstorf I have one outstanding quesiton about the functionality to be added here.  Will you update with what you intended?